### DISCUSSION TITLE
WebGPU backend

### DISCUSSION THREAD

**Discussion by davidroman0O** [AllenDang/cimgui-go#117] [Q&A]
Hello everyone

Simple question on the WebGpu backend support, is there any plan to integrate it? :) 
It would be nice to have both imgui and [webgpu](https://github.com/rajveermalviya/go-webgpu) easily

Great repo btw

**Comment by gucio321** [comment]
@davidroman0O unfortunately go is unable to compile cgo for wasm yet

**Reply by davidroman0O** [reply]
I understand that for the web part 
For the desktop part, it does :) 

Take a look at https://github.com/rajveermalviya/go-webgpu which uses https://github.com/gfx-rs/wgpu-native

I'm sorry i wasn't explicit enough in my first post! My bad! :)
